pkg/sdkserver/localsdk.go

```diff
@@ -425,3 +429,30 @@ func (l *LocalSDKServer) setGameServerFromFilePath(filePath string) error {
 	l.gs = convert(&gs)
 	return nil
 }
+
+/* Alpha Functionality */
```

Comment:
Can you also add an empty block / area for beta functionality?

Looking at this, it's going to be easy to miss graduating the blocks here from alpha -> beta -> ga when features move. Is there part of the feature documentation that needs to cover moving the code in this file?

Reply:
> Looking at this, it's going to be easy to miss graduating the blocks here from alpha -> beta -> ga when features move. Is there part of the feature documentation that needs to cover moving the code in this file?

This is a good point. Rather than having a block like this, does it make sense to make sure that "Alpha" is somehow in the comment block for the function, which means (a) we don't have to move it and (b) it should be much easier to pick up on? (or will it? :thinking:)

So rather than

```go
/* Alpha Functionality */

// PlayerConnect should be called when a player connects.
func (l *LocalSDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerId) (*alpha.Empty, error) {
	panic("implement me")
}
```

We would have

```go
// PlayerConnect should be called when a player connects.
// (Alpha)
func (l *LocalSDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerId) (*alpha.Empty, error) {
	panic("implement me")
}

// PlayerConnect should be called when a player connects.
// (Alpha)
func (l *LocalSDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerId) (*alpha.Empty, error) {
	panic("implement me")
}
```

etc? WDYT?

Might still require a change to the Feature Stage docs too though.

Reply:
Or maybe we should have some kind of Alpha + FeatureFlag combo?

```go
// PlayerConnect should be called when a player connects.
// (Alpha:PlayerTesting)
func (l *LocalSDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerId) (*alpha.Empty, error) {
	panic("implement me")
}
```

Other options:
- `[Alpha:PlayerTesting]`
- `Alpha:PlayerTesting`

Make it easy to search and replace for specific Alpha functionality. I specifically didn't put a space in, just because it's easy to get inconsistent with spacing.

Reply:
Putting it into each function makes sense. Then we can also put the functions into the file where we want them to live permanently (assuming they graduate out of alpha) instead of ending up with everything just being tacked onto the end and/or shuffling the blocks of code around during promotion (which can lead to missed changes during code review).

For the comment annotation, it's more verbose but I think it might be nice to do something like

```
// [Stage:Alpha]
// [FeatureFlag:PlayerTesting]
```

Since it will be more apparent to new developers how to emulate it without needing to look for documentation outside of the code.

Reply:
I like that a lot. Let's run with this :smile: